doc/configuration/logging.md

```diff
@@ -24,15 +20,18 @@ The following options are available in the `log` section:
 
 ## Example
 
-Multiple logging backends are supported.
+A single configurable backend is supported.
 
 ```yaml
 log:
   - output: stdout
     level:  trace
     format: plain
+```
+
+```yaml
```

Comment:
One last question: I understand that doing that will break the configuration format and probably many tests. But shouldn't it be more obvious to do the following since we allow only one backend (note there is no list of backends, all configs are top-level):

```yaml
log:
  output: stdout
  level: trace
  format: plain
```

Reply:
It would, but like you said it will break many tests etc. Perhaps we should create a new task, with this change, and test/code cleanup for it?

Reply:
Yeah, I think so. Please create a separate issue for that. Doing it here will make this PR too big.